hedera-node/hedera-app/src/main/java/com/hedera/node/app/throttle/ThrottleAccumulator.java

```diff
@@ -186,6 +189,36 @@ public boolean checkAndEnforceThrottle(
         return false;
     }
 
+    /**
+     * Determines if there is capacity in throttles that should be checked after the transaction handler has been called.
+     * Currently only relevant for contract ops duration throttle.
+     *
+     * @param txnInfo the transaction information
+     * @param txnStream the StreamBuilder containing the results of executing the transaction
+     * @param now the instant of time the transaction throttling should be checked for
+     * @return whether the transaction should be throttled
+     */
+    public boolean checkAndEnforcePostHandleThrottle(
```

Comment:
This may be a dumb thing to bring up, but how much work would it be to remove the 'post' concept from this API? In other words, is it possible to construct this post-handle throttle in such a way that the throttle doesn't need to know the operating context is past the transaction handling dispatch point?

Reply:
removed and replaced with `checkAndEnforceOpsDurationThrottle` as we will be checking the throttle during transaction processing